pkg/commands/cloud_config_import.go

```diff
@@ -170,11 +172,35 @@ func CloudTestRun(cmd *cloudTestCmd) {
 
 func performImport(testConfig *config.CloudTestConfig) error {
 	for _, imp := range testConfig.Imports {
+		if strings.HasSuffix(imp, "*") {
```

Comment:
May be better to have a full scope wildcards? replace all with .* and match as regular expression?